r/tests/testthat/test-dplyr-funcs-datetime.R

```diff
@@ -80,6 +80,25 @@ test_that("extract month from timestamp", {
       collect(),
     test_df
   )
+
+  skip_on_os("windows") # https://issues.apache.org/jira/browse/ARROW-13168
+
+  compare_dplyr_binding(
+    .input %>%
+      # R returns ordered factor whereas Arrow returns character
+      mutate(x = as.character(month(datetime, label = TRUE))) %>%
+      collect(),
+    test_df,
+    ignore_attr = TRUE
+  )
+
+  compare_dplyr_binding(
+    .input %>%
+      mutate(x = as.character(month(datetime, label = TRUE, abbr = TRUE))) %>%
+      collect(),
+    test_df,
+    ignore_attr = TRUE
+  )
```

Comment:
Would it be useful/helpful/important to test a date column too?

Reply:
It would; I will!

Reply:
Done